resources/styles/components/task-step/ends.less

```diff
@@ -11,6 +11,15 @@
   .completed-message {
     .tutor-completed-message();
     .size-exercise-card();
+
+		ul li {
```

Comment:
/me wonders if we should include a `.editorconfig` file... http://editorconfig.org/